src/shared/utilities/timeoutUtils.ts

```diff
@@ -27,38 +31,48 @@ export class Timeout {
      * Bottoms out at 0
      */
     public get remainingTime(): number {
-        const remainingTime = this._endTime - new Date().getTime()
+        const remainingTime = this.endTime - Date.now()
 
         return remainingTime > 0 ? remainingTime : 0
     }
 
+    /**
+     * Updates the timer to timeout in timeout length from now
```

Comment:
more familiar from other APIs, maybe it's clearer in general:
```suggestion
     * Resets elapsed time to zero
```

Reply:
oops cancel that, I see that "elapsed time" remains unchanged, which makes sense.

Reply:
Yep elapsed is only used for telemetry and tests, but in general it makes sense for what this class should do